### ISSUE TITLE
Format specifier doesn't match argument type

### ISSUE THREAD

**Issue by manuel-uberti** [dengste/org-caldav#73]
Hi,

this my org-caldav setup:
```
(use-package org-caldav ; Sync CalDAV calendars with Org
  :ensure t
  :defer t
  :config
  (setq org-caldav-calendars
        '((:calendar-id "boccaperta_shared_by_admin"
                        :url
                        "https://192.168.1.50/owncloud/remote.php/caldav/calendars/manuel/"
                        :inbox "~/org/calendars/from-boccaperta.org"
                        :files ("~/org/calendars/boccaperta.org"))
          (:calendar-id "extra_boccaperta_shared_by_giulia"
                        :url
                        "https://192.168.1.50/owncloud/remote.php/caldav/calendars/manuel/"
                        :inbox "~/org/calendars/from-extra-boccaperta.org"
                        :files ("~/org/calendars/extra-boccaperta.org")))
                   org-icalendar-timezone "Europe/Rome"))
```
Every time I try ```M-x org-caldav-sync```, this is what happens:
```
Debugger entered--Lisp error: (error "Format specifier doesn't match argument type")
  format("%02d:%02d" "nil" nil)
  icalendar--datetime-to-colontime(nil)
  org-caldav-convert-event()
  org-caldav-update-events-in-org()
  org-caldav-sync-calendar((:calendar-id "boccaperta_shared_by_admin" :url "https://192.168.1.50/owncloud/remote.php/caldav/calendars/manuel/" :inbox "~/org/calendars/from-boccaperta.org" :files ("~/org/calendars/boccaperta.org")) t)
  org-caldav-sync()
  funcall-interactively(org-caldav-sync)
  call-interactively(org-caldav-sync record nil)
  command-execute(org-caldav-sync record)
  helm-M-x(nil "org-caldav-sync")
  funcall-interactively(helm-M-x nil "org-caldav-sync")
  call-interactively(helm-M-x nil nil)
  command-execute(helm-M-x)
```
Any ideas?

**Comment by manuel-uberti** [comment]
No luck on this one?

**Comment by dengste** [comment]
Should be fixed by fd189bc5993bcb07f51a64f6f27728b9775a7af6.
